config/src/queue.rs

```diff
@@ -4,7 +4,7 @@ use iroha_config_base::derive::{Documented, Proxy};
 use serde::{Deserialize, Serialize};
 
 const DEFAULT_MAX_TRANSACTIONS_IN_QUEUE: u32 = 2_u32.pow(16);
-const DEFAULT_MAX_TRANSACTIONS_IN_SIGNATURE_BUFFER: u32 = 2_u32.pow(16);
+const DEFAULT_MAX_TRANSACTIONS_IN_QUEUE_PER_USER: u32 = 2_u32.pow(16);
```

Comment:
By setting this option to be equal to `DEFAULT_MAX_TRANSACTIONS_IN_QUEUE` we preserve old behavior. 
Not to break tests and benchmarks which can rely on a lot of transactions submitted by single user.